### PR TITLE
added UIActivityExtensions

### PR DESCRIPTION
**Pull request by staffler-xyz**
UIActivty.ActivityType has various constants for different services (see https://developer.apple.com/documentation/uikit/uiactivity/activitytype). Unfortunately, some important services are missing (WhatsApp, LinkedIn, etc.). This adds some of them with the possibility to add more in the future. Let me known what you think about this 😀👍

## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ### Added
 - **Dictionary**
   - Added `Dictionary[path:]` subscript for deep fetching/setting nested values. [#574](https://github.com/SwifterSwift/SwifterSwift/pull/573) by [@calebkleveter](https://github.com/calebkleveter)
+- **UIActivity**
+  - Added `ActivityType` constants for iCloud Drive, WhatsApp, LinkedIn, XING and Google Plus. [#580](https://github.com/SwifterSwift/SwifterSwift/pull/580) by [staffler-xyz](https://github.com/staffler-xyz)
```
Comment (guykogus) [inline CHANGELOG.md]: I don't see the Google+ addition. Also, isn't it being discontinued or something?

File: Sources/Extensions/UIKit/UIActivityExtensions.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public static let postToWhatsApp = UIActivity.ActivityType("net.whatsapp.WhatsApp.ShareExtension")
+
+    /// SwifterSwift: LinkedIn share extension
+    public static let postToLinkedIn = UIActivity.ActivityType("com.linkedin.LinkedIn.ShareExtension")
+
+    /// SwifterSwift: XING share extension
+    public static let postToXing = UIActivity.ActivityType("com.xing.XING.Xing-Share")
+}
+
+#endif
```
Comment (guykogus) [inline Sources/Extensions/UIKit/UIActivityExtensions.swift]: Add an empty line at the end.

**Comment by guykogus** [comment]
If you have a link to where you're getting these identifiers please add that in somewhere.

**Comment by staffler-xyz** [comment]
hmm, the most complete list I could find is this: http://johnszumski.com/blog/excluding-third-party-apps-from-ios-8-share-sheet (I don't know if these identifiers are all still valid)

I found the ones I listed by actually selecting and testing them in the `UIActivityController`. If you want, I could test some more if they are still valid!

File: Sources/Extensions/UIKit/UIActivityExtensions.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//  Created by Hannes Staffler on 07.10.18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(UIKit) && os(iOS)
+import UIKit
+
+// MARK: - ActivityType
+public extension UIActivity.ActivityType {
+    /// SwifterSwift: AddToiCloudDrive
```
Comment (guykogus) [inline Sources/Extensions/UIKit/UIActivityExtensions.swift]: Add an empty line above here

**Comment by guykogus** [comment]
If you want you can test all of them, but honestly for something like this I'd say just add the ones that you're using for your project. If somebody needs more they can add at a later stage. Probably best to put that link in a comment in the code so that it can be referenced in the future.

**Comment by guykogus** [comment]
@staffler-xyz please resolve the conflicts and fix the (very) minor change requests

**Pull request merged**